make/migrations/postgresql/0030_1.11.0_schema.up.sql

```diff
@@ -43,6 +43,19 @@ CREATE TABLE artifact_reference
   CONSTRAINT  unique_reference UNIQUE (parent_id, child_id)
 );
 
+/* artifact_trash records deleted artifact */
+CREATE TABLE artifact_trash
+(
+  id            SERIAL PRIMARY KEY NOT NULL,
+  artifact_id   int NOT NULL,
```

Comment:
Why we need to record the artifact ID here?

Reply:
could be removed